mne/filter.py

```diff
@@ -1534,6 +1534,14 @@ def savgol_filter(self, h_freq):
             done using polynomial fits instead of FIR/IIR filtering.
             This parameter is thus used to determine the length of the
             window over which a 5th-order polynomial smoothing is used.
+        copy : bool
```

Comment:
I don't know if it matters or not but the copy description is a bit different given the function. should we have search/replace? we can also leave it as is